### PR TITLE
Javascript/javascript2/week2

### PR DESCRIPTION
**Pull request by zmzade**

### CODE REVIEW

File: javascript/javascript2/week2/movies.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+const movies = require("./movies.json");
+const shortTitleMovies = movies.filter((x) => x.title.length < 3);
+//console.log(shortTitleMovies);
+const longTitleMovies = movies.filter((x) => x.title.length > 50);
+//console.log(longTitleMovies);
+
+const moviesIn80s = movies.filter((x) => (x.year >= 1980) & (x.year <= 1989));
```
Comment (calm-prog) [inline javascript/javascript2/week2/movies.js]: `&` is a bitwise operator, and it has its special use case. I think you wanted to use the standard AND operator which is `&&`. Correct me if I am wrong.

File: javascript/javascript2/week2/main.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      (country) => (shipsToHTML += `<li>${country}</li>`)
+    );
+
+    li.innerHTML = `
+            <ul>
+                <li>${product.name}</li>
+                <li>${product.price}</li>
+                <li>${product.rating}</li>
+                <ul class="ships-to">${shipsToHTML}</ul>
+            </ul>
```
Comment (calm-prog) [inline javascript/javascript2/week2/main.js:23]: Unordered list contains elements of the same type. You could use just divs here or make it a [table](https://www.w3schools.com/html/html_tables.asp).

**Review by calm-prog — COMMENTED** [review]
It looks very good! :+1:  I have only minor comments.

File: javascript/javascript2/week2/movies.js (hunk 1 above)
Comment (zmzade) [reply javascript/javascript2/week2/movies.js]: You are completely right.

File: javascript/javascript2/week2/main.js (hunk 2 above)
Comment (zmzade) [reply javascript/javascript2/week2/main.js:23]: I think it is  inside homework, I just copied it. It is not mine. Any way thank you 👍

**Pull request merged**